### ISSUE TITLE
GTC file version 3 can't output B allele frequencies and logr

### ISSUE THREAD

**Issue by Liiiii2101** [Illumina/BeadArrayFiles#20]
Thank you so much for making this repo. The B allele frequencies and logr values are important features for our studies but the version of our GTC files is 3, which means we cannot get those two features from your library. Do you know any other way to get these two features or the way to convert idat files to gtc version 4 or 5.

Thank you very much!

**Comment by jjzieve** [comment]
@Liiiii2101 You can genotype (i.e. IDAT->GTC conversion) via https://emea.support.illumina.com/array/array_software/illumina-array-analysis-platform.html 
which should create GTC version 5 with B Allele Frequencies and LogRRatios

**Comment by Liiiii2101** [comment]
> @Liiiii2101 You can genotype (i.e. IDAT->GTC conversion) via https://emea.support.illumina.com/array/array_software/illumina-array-analysis-platform.html
> which should create GTC version 5 with B Allele Frequencies and LogRRatios

Thank you! I used iaap to convert the idat files to gtcs, but there's another issue that the gtc files are incomplete. Do you know any methods to solve it?

Thank you very much!

**Comment by jjzieve** [comment]
I would contact illumina support directly (techsupport@illumina.com) for issues related to iaap, there may be a workaround.

**Comment by jjzieve** [comment]
@Liiiii2101 Was this ever resolved?

**Comment by mikaelamo** [comment]
I am having the same issue. Did any of you find a solution?

**Comment by jjzieve** [comment]
@mikaelamo Did you try running iaap? Did you get the same gtc incomplete error?

**Comment by mikaelamo** [comment]
@jjzieve The GTC files I have were created by using iaap, which is why I feel that it's kinda weird that they should be incomplete since iaap is supposed to be compatible with BeadArrayFiles. So far, I have only used the example code from the README.md, to see if I can get anything to work before deep diving but alas...

**Comment by jjzieve** [comment]
@mikaelamo Yes in general, iaap should be compatible. But because the original GTCs were version 3, I'd assume the product you're running is pretty old. So something untested/expected might be happening. Its hard to say for sure what's going on. I  recommend the same path I recommended @Liiiii2101. To reach out to [techsupport@illumina.com](mailto:techsupport@illumina.com) so we can open a support channel where we can take a look at your GTCs and/or IDATs and debug more closely. I don't remember if @Liiiii2101 ever reached out to support but it seems I asked about it and never got a response. Hopefully if we find out the root cause of your issue, the findings can be helpful to others here.
